src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -454,7 +518,7 @@ namespace
             Spectrum&               value,
             float&                  probability) const
         {
-            outgoing = -normalize(light_transform.get_parent_z());
+            outgoing = -normalize(light_transform.get_parent_z() + light_transform.get_parent_origin());
```

Comment:
Not sure to understand what's going on here. Seems wrong.

Reply:
This was to shift the sun but should be done after we compute sun radiance.